deploy/examples/ApplicationMonitoring.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: applicationmonitoring.integreatly.org/v1alpha1
 kind: ApplicationMonitoring
 metadata:
   name: example-applicationmonitoring
+spec:
+  namespaceSelector: "middleware"
```

Comment:
I think we should call this just `selector` because it's not only used for namespaces (the prometheus operator also checks that label for service monitors and rules)